### PR TITLE
Feature/hsr bpm

### PR DESCRIPTION
**Pull request by sylar-zeyangz**

### CODE REVIEW

File: icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                        (SubmitHealthServiceRequestResponse)
+                                webServiceTemplate.marshalSendAndReceive(
+                                        hsrServiceUrl, submitHealthServiceRequest);
+            } catch (WebServiceIOException ex) {
+                // Connection Error
+                appErr = false;
+                hsrFail = true;
+                break;
+            } catch (Exception ex) {
+                if (++retries == 5) {
```
Comment (ekwanNTT) [inline icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java:74]: Should this 5 be MAX_RETRIES?

File: icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                    appErr = true;
+                    hsrFail = true;
+                    break;
+                }
+                Thread.sleep(5000);
```
Comment (ekwanNTT) [inline icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java:79]: Please extract to a constant

**Review by ekwanNTT — CHANGES_REQUESTED** [review]

File: icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java (hunk 1 above)
Comment (sylar-zeyangz) [reply icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java:75]: yes!

Comment (sylar-zeyangz) [reply icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java]: fixed

File: icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java (hunk 2 above)
Comment (sylar-zeyangz) [reply icon-two-rabbit-consumer/src/main/java/ca/bc/gov/open/jagiconconsumer/services/HSRService.java]: fixed

**Review by ekwanNTT — APPROVED** [review]

**Pull request merged**